contents/docs/product-analytics/tutorials.mdx

```diff
@@ -14,6 +14,7 @@ Got a question which isn't answered below? Head to [the community forum](/questi
 - [How to set up cross domain tracking](/tutorials/cross-domain-tracking)
 - [How to track pageviews in single-page apps (SPA)](/tutorials/single-page-app-pageviews)
 - [How to create actions using CSS selectors](/tutorials/css-selectors-for-actions)
+- [How to rename events](/tutorials/how-to-rename-events)
 
```

Comment:
I think we should add this as a question to the FAQ: https://posthog.com/docs/product-analytics/troubleshooting

the answer should be something like: "you should use actions. Here's a tutorial which shows in depth"